xcell/mappers/mapper_2MPZ.py

```diff
@@ -111,6 +110,8 @@ def get_signal_map(self, apply_galactic_correction=True):
 
     def get_mask(self):
         if self.mask is None:
+            # We will assume the mask has been provided in the right
+            # coordinates, so no further conversion is needed.
```

Comment:
Right, so for both WIxSC and 2MPZ the catalogs and the masks are provided both in galactic and equatorial coordinates. So it seems silly to me that we rotate them.
I guess one option if we want to ensure that we're reading the right file would be to call the mask parameter "mask_{self.coords}" instead of "mask", so an error will be thrown if we haven't provided that explicitly.